Pipfile

```diff
@@ -13,6 +13,8 @@ ipython = "*"
 faker = "*"
 factory-boy = "*"
 pylint-django = "*"
+pandas = "*"
+jupyter = "*"
```

Comment:
also you were right that jupyter is awesome, it was nice to be able to easily re-run parts of the big script